jobs/build_ova/prepare_ova_post_test.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash +xe
+set -x
+OVA=`ls ${OVA_PATH}`
+
+echo "Post Test starts "
+
+deployOva() {
+    echo yes | ovftool \
+    --overwrite --powerOffTarget --powerOn --skipManifestCheck \
+    --net:"CONTROL=${NODE_NAME}-switch" \
+    --datastore=${DATASTORE} \
+    --name=${NODE_NAME}-ova-for-post-test \
+    ${OVA} \
+    "vi://${VCENTER_NT_USER}:${VCENTER_NT_PASSWORD}@${VCENTER_IP}/${VCENTER_PATH}/${ESXI_HOST}"/
+    if [ $? = 0 ]; then
+        echo "[Info] Deploy OVA successfully".
+    else
+        echo "[Error] Deploy OVA failed."
+        exit 3
+    fi
+    ssh-keygen -f "$HOME/.ssh/known_hosts" -R $OVA_INTERNAL_IP
```

Comment:
OVA_INTERNAL_IP never changes but the OVA always change.

Error will occur when ssh the same IP of different fingerprint.